cleanup-oracle.sh

```diff
@@ -192,12 +192,11 @@ fi
 
 export ANSIBLE_NOCOWS=1
 
-ANSIBLE_PLAYBOOK=`which ansible-playbook 2> /dev/null`
+ANSIBLE_PLAYBOOK="ansible-playbook"
+ANSIBLE_PLAYBOOK_CHECK=`ansible-playbook --version 2> /dev/null`
 if [ $? -ne 0 ]; then
     echo "Ansible executable not found in path"
     exit 3
-else
-    echo "Found Ansible at $ANSIBLE_PLAYBOOK"
```

Comment:
I think we're losing something here:  on a system with multiple ansible installs, it can be very useful to see in the log which one was used.  A few non-`which` ideas:

https://unix.stackexchange.com/questions/85249/why-not-use-which-what-to-use-then

Reply:
Thanks, this is a terrific link. Good to know the history.
I think we can go with `type`:
```bash
jcnarasimhan@jon2:~/DriveFS/My Drive/bmaas/test_automation/oss_prow_onboarding/running_cleanup_install_oracle$ kubectl exec -it test-pods-namespace-lo-podname -n test-pods -- /bin/bash
bash-4.4# type ansible-playbook
ansible-playbook is /usr/local/bin/ansible-playbook
bash-4.4# exit
exit
jcnarasimhan@jon2:~/DriveFS/My Drive/bmaas/test_automation/oss_prow_onboarding/running_cleanup_install_oracle$ type ansible-playbook
ansible-playbook is hashed (/usr/bin/ansible-playbook)
```

Reply:
For this change:
```bash
ANSIBLE_PLAYBOOK="ansible-playbook"
if ! type ansible-playbook > /dev/null 2>&1; then
  echo "Ansible executable not found in path"
  exit 3
else
  echo "Found Ansible: `type ansible-playbook`"
fi
```

The output is as follows:
```
bash-4.4# ./cleanup-oracle.sh --ora-version 19 --inventory-file /etc/files_needed_for_tk/nonrac-inv --yes-i-am-sure --ora-disk-mgmt udev --ora-swlib-path /u01/oracle_install --ora-asm-disks /etc/files_needed_for_tk/nonrac-asm.json --ora-data-mounts /etc/files_needed_for_tk/nonrac-datamounts.json
Command used:
./cleanup-oracle.sh --ora-version 19 --inventory-file /etc/files_needed_for_tk/nonrac-inv --yes-i-am-sure --ora-disk-mgmt udev --ora-swlib-path /u01/oracle_install --ora-asm-disks /etc/files_needed_for_tk/nonrac-asm.json --ora-data-mounts /etc/files_needed_for_tk/nonrac-datamounts.json

Running with parameters from command line or environment variables:

INVENTORY_FILE=/etc/files_needed_for_tk/nonrac-inv
ORA_ASM_DISKS=/etc/files_needed_for_tk/nonrac-asm.json
ORA_DATA_MOUNTS=/etc/files_needed_for_tk/nonrac-datamounts.json
ORA_DISK_MGMT=udev
ORA_ROLE_SEPARATION=TRUE
ORA_STAGING=/u01/oracle_install
ORA_SWLIB_PATH=/u01/oracle_install
ORA_VERSION=19.3.0.0.0

Ansible params: 
Found Ansible: ansible-playbook is /usr/local/bin/ansible-playbook

Running Ansible playbook: ansible-playbook -i /etc/files_needed_for_tk/nonrac-inv  brute-cleanup.yml

PLAY [dbasm] *******************************************************************************************************************************************************************

TASK [Gathering Facts] *********************************************************************************************************************************************************
```